consensus/src/marshal/mod.rs

```diff
@@ -0,0 +1,336 @@
+//! Ordered notification of finalized blocks.
```

Comment:
nit: `Ordered delivery of finalized blocks.`

Could also note it is "at least once" (or that the caller should be prepared to process things twice)?

Reply:
I'll note the at-least-once delivery below (rather than in this header one-liner)